modules/benchmarks/src/tree/iv/tree.js

```diff
@@ -5,5 +5,5 @@
){return e.toString()}function charValues(e){return String.fromCharCode("A".charCodeAt(0)+e%26)}function refresh(e){renderer||(renderer=htmlRenderer(document.getElementById("root"),tree)),renderer.refresh({data:e})}function destroyDom(){refresh(emptyTree)}function createDom(){refresh(data=buildTree())}function detectChanges(){for(var e=0;e<10;e++)refresh(data);detectChangesRuns+=10,numberOfChecksEl.textContent=""+detectChangesRuns}function getColor(e){return e%2?"":"grey"}function tree(e){var n,r,t,o,i=$iv.node,d=e.data;i.cm?(r=$el(n=$el(i,1,"tree",1),2,"span",0),$cm("style","backgroundColor",0,0,getColor(d.depth),r),$dt(r,3,""+$ct(d.value))):(t=(r=(n=i.children[0]).children[0]).children[0],$ut(""+$ct(d.value),t,i)),o=1,null!=d.left&&(r=$cg(o,n,i,4),o++,0,r.cm?(t=$cc(r,5,{data:d.left},tree,0,1),r.cm=0):(t=r.children[0],$uc("data",d.left,t),$rc(t,r)),$lg(r,i)),null!=d.right&&($dg(o,n,i,6),r=$cg(o,n,i,6),o++,0,r.cm?(t=$cc(r,7,{data:d.right},tree,0,1),r.cm=0):(t=r.children[0],$uc("data",d.right,t),$rc(t,r)),$lg(r,i)),i.cm?i.cm=0:$dg(o,n,i,8)}function noop(){}var EMPTY_PROPS={},NO_PROPS=void 0,refreshCount=0,refCount=0,$iv={node:null,renderer:null,reset:function(e){1===e?refCount=0:2===e&&(refreshCount=0)},get refreshCount(){return refreshCount}},RX_HTML=/html/i,Renderer=function(){function e(e,n,r){this.htmlElement=e,this.vdFunction=n;var t=this.vdom={kind:"G",index:0,ref:$nextRef(),cm:1,changes:null,childChanges:null,children:[],domNode:null,parent:null,$lastChange:$iv.refreshCount,$lastRefresh:$iv.refreshCount},o={kind:"CG",node:t,parent:null,position:-1,nextSibling:null};t.changes=[o],this.doc=r||{createDocFragment:function(){return document.createDocumentFragment()},createTextNode:function(e){return document.createTextNode(e)},createElement:function(e){return document.createElement(e)},createElementNS:function(e,n){return document.createElementNS(e,n)}}}return e.prototype.refresh=function(e){$refreshTemplate(this,this.vdFunction,this.vdom,e)},e.prototype.processChanges=function(e){processChanges(e,this.htmlElement,this.doc)},e}();urlParamsToForm();var TreeNode=function(){function e(e,n,r,t,o){this.value=e,this.depth=n,this.maxDepth=r,this.left=t,this.right=o,this.transitiveChildCount=Math.pow(2,this.maxDepth-this.depth+1)-1,this.children=this.left?[this.left,this.right]:[]}return Object.defineProperty(e.prototype,"style",{get:function(){return this.depth%2==0?"background-color: grey":""},enumerable:!0,configurable:!0}),e}(),treeCreateCount,maxDepth,numberData,charData;init();var emptyTree=new TreeNode("",0,0,null,null),renderer,data=emptyTree,numberOfChecksEl=document.getElementById("numberOfChecks"),detectChangesRuns=0;bindAction("#destroyDom",destroyDom),bindAction("#createDom",createDom),bindAction("#detectChanges",detectChanges),bindAction("#detectChangesProfile",profile(detectChanges,noop,"detectChanges")),bindAction("#updateDomProfile",profile(createDom,noop,"update")),bindAction("#createDomProfile",profile(createDom,destroyDom,"create"));
```

Comment:
I don't know how I feel about checking in minified code. Thoughts?

Reply:
This is generated code from the original IV prototype. It is meant to be used as a reference and is not supposed to change (now that it is fixed with this PR).

I don't think it would be worth the effort to add the full prototype in the repo and generating this code as a build step.